### DISCUSSION TITLE
GUI Like PHPMYADMIN

### DISCUSSION THREAD

**Discussion by RAIESHG** [planetscale/discussion#264] [Feature Requests] [answered]
So it would be better if we not only had the command line on the website for our database visualization and manipulation, it would be much better if there is an userinterface like PHPMYADMIN for users to interact with their data in the database.

**Comment by fooforge** [comment, marked as answer]
@RAIESHG Thanks for the feedback! I'll make an internal note.

Check out [MySQL Workbench](https://dev.mysql.com/downloads/workbench/) in the mean time. That should get you what you're after.

**Reply by step135** [reply]
Any news when you will support it?

**Comment by ConstantinFF** [comment]
@RAIESHG You can use docker phpmyadmin

```
# docker-compose.yml
services:
  phpmyadmin:
    image: phpmyadmin
    ports:
      - "80:80"
    volumes:
      - ./config.user.inc.php:/etc/phpmyadmin/config.user.inc.php
    environment:
      PMA_HOST: PLANETSCALE_HOST
      PMA_USER: PLANETSCALE_USERNAME
      PMA_PASSWORD: PLANETSCALE_PASSWORD  
```
```
<?php
// config.user.inc.php
$i++;
$cfg['Servers'][$i]['ssl_ca'] = '/etc/ssl/certs/ca-certificates.crt';
$cfg['Servers'][$i]['ssl']=true;
```
